ext/hal/st/stm32cube/stm32f0xx/README

```diff
@@ -1,15 +1,48 @@
-The current version supported in Zephyr for STM32F0 Cube is V1.9.0
+STM32CubeF0
+###########
+
+Origin:
+   ST Microelectronics
+   http://www.st.com/en/embedded-software/stm32cubef0.html
+
+Status:
+   version 1.9.0
+
+Purpose:
+   ST Microelectronics official MCU package for STM32F0 series.
+
+Description:
+   This package is an extract of official STM32CubeF0 package written by ST Microelectronics.
+   It is composed of STM32Cube hardware abstraction layer (HAL) and low layer (LL) plus a set
+   of CMSIS headers files, one for each SoC in STM32F0 series.
+
+Dependencies:
+    None.
+
+URL:
+   http://www.st.com/en/embedded-software/stm32cubef0.html
+
+commit:
```

Comment:
Does this make sense in stm32cubefx case? The project is not hosted on a git repository or something similar.

Reply:
Not much but copy/pasted from similar library (with also absence of git repo),
and README reported form that library was reported compliant with Zephyr guidelines,
hence I reused it to minimize chances to get a comment to rework it.